src/commands/Info.js

```diff
@@ -0,0 +1,14 @@
+const BaseCommand = require('../structures/BaseCommand');
+
+class Ping extends BaseCommand {
+  constructor() {
+    super('info');
+  }
+
+  async run(client, message) {
+      message.channel.send('This bot was made by The Mauve-Development Teamn/Xa_Puppet & Jonfirexbox')
+    });
```

Comment:
* Check bracket placement, and exports class

Reply:
@Wilbert-mad idk what i have to change?

Reply:
@Jonfirexbox Check the class you are exporting and the class name. lastly, check the bracket placement on line 10.
and I'll review this request again please and thank you.

Reply:
Ok